stdlib/public/Concurrency/Clock.cpp

```diff
@@ -134,6 +157,20 @@ switch (clock_id) {
       *nanoseconds = suspending.tv_nsec;
       return;
     }
+    case swift_clock_id_wall: {
+      struct timespec wall;
+#if defined(__linux__) || defined(__APPLE__) || defined(__OpenBSD__) || defined(__FreeBSD__) || defined(__wasi__)
+      clock_getres(CLOCK_REALTIME, &wall);
```

Comment:
As above, `timespec_getres(&wall, TIME_UTC)` is standard and works on Windows too.